dali/test/python/test_operator_erase.py

```diff
@@ -189,42 +183,54 @@ def test_operator_erase_vs_python():
+                                                anchor_norm, shape_norm, normalized_anchor,
+                                                normalized_shape):
     eii1 = RandomDataIterator(batch_size, shape=input_shape, dtype=np.float32)
     eii2 = RandomDataIterator(batch_size, shape=input_shape, dtype=np.float32)
     compare_pipelines(
         ErasePipeline(device, batch_size, input_layout, iter(eii1),
                       anchor=anchor_norm, shape=shape_norm,
                       normalized_anchor=normalized_anchor, normalized_shape=normalized_shape,
                       axis_names=axis_names, axes=None, fill_value=0),
-        ErasePipeline(device, batch_size, input_layout, iter(eii1),
+        ErasePipeline(device, batch_size, input_layout, iter(eii2),
                       anchor=anchor, shape=shape, axis_names=axis_names, axes=None, fill_value=0),
         batch_size=batch_size, N_iterations=3, eps=1e-04)
 
 
 def test_operator_erase_with_normalized_coords():
     # layout, shape, axis_names, anchor, shape, anchor_norm, shape_norm
-    rois = [("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), 0),
-            ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), -1),
-            ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), (118, 186, 0)),
-            ("HWC", (60, 80, 3), "HW", (4, 2, 3, 4), (50, 10, 10, 50), (4 / 60.0, 2 / 80.0, 3 / 60.0, 4 / 80.0), (50 / 60.0, 10 / 80.0, 10 / 60.0, 50 / 80.0), 0),
-            ("HWC", (60, 80, 3), "H", (4,), (7,), (4 / 60.0,), (7 / 60.0,), 0),
-            ("DHWC", (10, 60, 80, 3), "DHW", (2, 4, 10), (5, 40, 50), (2 / 10.0, 4 / 60.0, 10 / 80.0), (5 / 10.0, 40 / 60.0, 50 / 80.0), 0),
-            ("HWC", (60, 80, 1), "WH", (10, 4), (50, 40), (10 / 80.0, 4 / 60.0), (50 / 80.0, 40 / 60.0), 0),
-            ("XYZ", (60, 80, 3), "X", (4,), (7,), (4 / 60.0,), (7 / 60.0,), -10),]
+    rois = [
+        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0),
+         0),
+        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0),
+         -1),
+        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50), (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0),
+         (118, 186, 0)),
+        ("HWC", (60, 80, 3), "HW", (4, 2, 3, 4), (50, 10, 10, 50),
+         (4 / 60.0, 2 / 80.0, 3 / 60.0, 4 / 80.0), (50 / 60.0, 10 / 80.0, 10 / 60.0, 50 / 80.0), 0),
+        ("HWC", (60, 80, 3), "H", (4, ), (7, ), (4 / 60.0, ), (7 / 60.0, ), 0),
+        ("DHWC", (10, 60, 80, 3), "DHW", (2, 4, 10), (5, 40, 50), (2 / 10.0, 4 / 60.0, 10 / 80.0),
+         (5 / 10.0, 40 / 60.0, 50 / 80.0), 0),
+        ("HWC", (60, 80, 1), "WH", (10, 4), (50, 40), (10 / 80.0, 4 / 60.0), (50 / 80.0, 40 / 60.0),
+         0),
+        ("XYZ", (60, 80, 3), "X", (4, ), (7, ), (4 / 60.0, ), (7 / 60.0, ), -10),
```

Comment:
This is terrible - I'd recommend reverting and adding `#noqa` or formatting by hand, e.g:
```suggestion
    rois = [
        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50),
         (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), 0),
        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50),
         (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), -1),
        ("HWC", (60, 80, 3), "HW", (4, 10), (40, 50),
         (4 / 60.0, 10 / 80.0), (40 / 60.0, 50 / 80.0), (118, 186, 0)),
        ("HWC", (60, 80, 3), "HW", (4, 2, 3, 4), (50, 10, 10, 50),
         (4 / 60.0, 2 / 80.0, 3 / 60.0, 4 / 80.0), (50 / 60.0, 10 / 80.0, 10 / 60.0, 50 / 80.0), 0),
        ("HWC", (60, 80, 3), "H", (4, ), (7, ), (4 / 60.0, ), (7 / 60.0, ), 0),
        ("DHWC", (10, 60, 80, 3), "DHW", (2, 4, 10), (5, 40, 50),
         (2 / 10.0, 4 / 60.0, 10 / 80.0), (5 / 10.0, 40 / 60.0, 50 / 80.0), 0),
        ("HWC", (60, 80, 1), "WH", (10, 4), (50, 40),
         (10 / 80.0, 4 / 60.0), (50 / 80.0, 40 / 60.0), 0),
        ("XYZ", (60, 80, 3), "X", (4, ), (7, ), (4 / 60.0, ), (7 / 60.0, ), -10),
```